tools/Linux/kodi.sh.in

```diff
@@ -25,16 +25,20 @@ prefix="@prefix@"
 exec_prefix="@exec_prefix@"
 datarootdir="@datarootdir@"
 LIBDIR="@libdir@"
+APP_BINARY=$LIBDIR/${bin_name}/@APP_BINARY@
 CRASHLOG_DIR=${CRASHLOG_DIR:-$HOME}
 USERDATA_DIR="${HOME}/.${bin_name}"
-
+WINDOWING=${WINDOWING:-"auto"}
 
 # Workaround for high CPU load with nvidia GFX
 export __GL_YIELD=USLEEP
 
 # Fix wasting RAM due to fragmentation
 export MALLOC_MMAP_THRESHOLD_=131072
 
+# Workaround for VAAPI with glvnd
+export LIBVA_DRIVER_NAME=${LIBVA_DRIVER_NAME:-"i965"}
```

Comment:
I think this would better fit in a separate commit since it's unrelated and so that we can revert it more easily. We will have to remove it again at some point.